<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>371</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>371</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>371</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>HAS SOHBET</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icon.qrc">
    <normaloff>:/img/img/h.png</normaloff>:/img/img/h.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(1, 0, 42);
	border-style: outset;
    border-width: 2px;
	border: 1px;
	border-radius: 10px;
	font: 75 bold 18pt &quot;Ubuntu&quot;;
	color: rgb(199, 133, 0);
}
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background-color: rgb(1, 0, 42);
	border-style: outset;
    border-width: 2px;
	border: 1px;
	border-radius: 10px;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../icon.qrc">:/img/img/h.png</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="birframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#birframe{
	background-color: rgb(87, 137, 139, 200);
	border: 1px solid white;
    border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="ikiframe">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>491</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ikiframe{
	background-color: rgb(255, 255, 255, 140);
	border: 1px;
    border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="kads">
           <property name="styleSheet">
            <string notr="true">#kads{
	color: rgb(0, 0, 0);
	font: 75 bold 10pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <property name="text">
            <string>HAS SOHBET'E HOŞGELDİNİZ</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLCDNumber" name="number">
           <property name="styleSheet">
            <string notr="true">#number{
	background-color: rgb(255, 255, 255, 100);
    border-style: outset;
    border-width: 2px;
	color: rgb(0, 0, 0);
	border: 1px;
	border-radius: 10px;
}</string>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>logincontrol()</slot>
 </slots>
</ui>
